<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDialog</class>
 <widget class="QDialog" name="ChangePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Change Password</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #2c3e50; margin-bottom: 4px;</string>
     </property>
     <property name="text">
      <string>üîê  Change Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblUser">
     <property name="styleSheet">
      <string notr="true">color: #7f8c8d; font-size: 12px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblOldPwd">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 12px;</string>
     </property>
     <property name="text">
      <string>Current password:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditOldPwd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Enter current password...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnToggleOld">
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
    background:#ecf0f1; 
    border-radius:6px; 
    border:1px solid #bdc3c7; 
    font-size:14px; 
}
QPushButton:hover { 
    background:#d5dbdb; 
}</string>
       </property>
       <property name="text">
        <string>üëÅ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblNewPwd">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 12px;</string>
     </property>
     <property name="text">
      <string>New password:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEditNewPwd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Minimum 6 characters...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnToggleNew">
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
    background:#ecf0f1; 
    border-radius:6px; 
    border:1px solid #bdc3c7; 
    font-size:14px; 
}
QPushButton:hover { 
    background:#d5dbdb; 
}</string>
       </property>
       <property name="text">
        <string>üëÅ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblConfirmPwd">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 12px;</string>
     </property>
     <property name="text">
      <string>Confirm new password:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditConfirmPwd">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Re-enter new password...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
    background:#2ecc71; 
    color:white; 
    border-radius:6px;
    font-size:13px; 
    font-weight:bold; 
    padding: 0 16px; 
}
QPushButton:hover { 
    background:#27ae60; 
}</string>
       </property>
       <property name="text">
        <string>üíæ  Save Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>38</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
    background:#ecf0f1; 
    color:#7f8c8d; 
    border-radius:6px;
    font-size:13px; 
    padding: 0 16px; 
}
QPushButton:hover { 
    background:#d5dbdb; 
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ChangePasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
